rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -94,6 +96,41 @@ class BridgeSerializationUtilsTest {
     private static final Address ADDRESS = BitcoinTestUtils.createP2PKHAddress(MAINNET_PARAMETERS, "first");
     private static final Address OTHER_ADDRESS = BitcoinTestUtils.createP2PKHAddress(MAINNET_PARAMETERS, "second");
 
+    private static Stream<Arguments> validRskAddressesProvider() {
+        return Stream.of(
+            Arguments.of(TestUtils.generateAddress("address1")),
+            Arguments.of(TestUtils.generateAddress("address2")),
+            Arguments.of(TestUtils.generateAddress("address3"))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("validRskAddressesProvider")
+    void serializeAndDeserializeRskAddress_validAddress_ok(RskAddress address) {
+        // Act & Assert
+        byte[] actualSerializedAddress = BridgeSerializationUtils.serializeRskAddress(address);
+        assertNotNull(actualSerializedAddress);
+
+        RskAddress actualDeserializedAddress = BridgeSerializationUtils.deserializeRskAddress(actualSerializedAddress);
+        assertNotNull(actualDeserializedAddress);
+
+        assertEquals(address, actualDeserializedAddress);
+    }
+
+    @Test
+    void serializeRskAddress_whenNull_shouldFail() {
+        // Act & Assert
+        assertThrows(IllegalArgumentException.class, () -> BridgeSerializationUtils.serializeRskAddress(null));
+    }
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    void deserializeRskAddress_whenInvalidValues_shouldReturnNull(byte[] nullOrEmptyData) {
```

Comment:
```suggestion
    void deserializeRskAddress_whenNullOrEmptyData_shouldReturnNull(byte[] data) {
```

i think is better to make the name more descriptive instead of the variable

Reply:
Done